scripts/jest/createCacheKeyFunction.js

```diff
@@ -7,34 +7,34 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 
-var crypto = require('crypto');
-var fs = require('fs');
-var path = require('path');
+ 'use strict';
+
+const crypto = require('crypto');
+const fs = require('fs');
+const path = require('path');
 
 function buildCacheKey(files, base) {
-  return files.reduce(function(src, fileName) {
-    return src + fs.readFileSync(fileName);
-  }, base);
+  return files.reduce(
+    (src, fileName) => src + fs.readFileSync(fileName),
+    base
+  );
 }
 
-var transformRoot = path.join(__dirname, '..');
-var cacheKeyFiles = [
+const transformRoot = path.join(__dirname, '..');
+const cacheKeyFiles = [
   __filename,
-  path.join(transformRoot, 'babel/default-options.js'),
-  path.join(transformRoot, 'babel/dev-expression.js'),
-  path.join(transformRoot, 'babel/inline-requires.js'),
-  path.join(transformRoot, 'babel/rewrite-modules.js'),
+  path.join(transformRoot, 'babel', 'default-options.js'),
+  path.join(transformRoot, 'babel', 'dev-expression.js'),
+  path.join(transformRoot, 'babel', 'inline-requires.js'),
+  path.join(transformRoot, 'babel', 'rewrite-modules.js'),
 ];
 
-var cacheKeyBase = buildCacheKey(cacheKeyFiles, '');
-
-module.exports = function(files) {
-  var cacheKey = buildCacheKey(files, cacheKeyBase);
+const cacheKeyBase = buildCacheKey(cacheKeyFiles, '');
 
-  return function(src, file, options, excludes) {
-    return crypto.createHash('md5')
-      .update(cacheKey)
-      .update(JSON.stringify([src, file, options, excludes]))
-      .digest('hex');
-  };
+module.exports = files => {
```

Comment:
Oh I lied when I said this could be a patch change. We'll bump minor. Using the arrow means iojs breaks (but jest 0.5 still worked there so somebody *could* be using it).